kubernetes/kserve/tests/configs/mnist_oip_http.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: "torchserve-mnist-v2-http"
+spec:
+  predictor:
+    pytorch:
+      protocolVersion: v2
+      storageUri: gs://kfserving-examples/models/torchserve/image_classifier/v2
+      image: sivanantha/torchserve-oip-test:latest
```

Comment:
Is it possible to use the latest built image for this test ? I am trying to use open inference protocol but the preprocess is not working